our-work/software/cogresnet.yml

```diff
@@ -0,0 +1,20 @@
+title: Cognitive Research Registration App
+description: "Firebase Application that allows participants to register and researchers to manage their registration"
+languages:
+  - javascript
```

Comment:
I think other projects have only listed `vuejs` - is the `javascript` needed?

Reply:
They all list `javascript` except for riddc ;) - so I added it to riddc. But I'm happy to remove it from all if needed. I think faculty may look for do we know javascript?